Major Update (#268)
NilPointer-Software/bdfd-wiki: src/bdscript/onlyAdmin.md

$onlyAdmin[errorMessage]
```

### Parameters 

> - `errorMessage` `(Type : String || Flag : Emptiable)` : The error message that is returned when the user doesn't have administrator.

Reviewer: ```suggestion
- `errorMessage` `(Type: String || Flag: Emptiable)`: The error message that is returned when the user isn't an administrator.
```
